setup.cfg

```
@@ -41,12 +41,13 @@ dev =
     pylint-django==2.0.13
     pylint==2.4.4
     pytest-cov==2.8.1
-    pytest-django==3.8.0
-    pytest==5.3.5
+    pytest-django==4.5.2
+    pytest==7.4.0
 ci =
     twine==2.0.0
 sandbox =
-    django-configurations==2.2
+    Django==4.2
```

```suggestion
    Django==4.2.4
```

The 4.2 is a little buggy, you should use the last version.